datasets/_externals/wikidata.yml

```diff
@@ -978,6 +979,8 @@ lookups:
           - South African Republic
           - Ciskei
         value: ZA
+      - match: Villavicencio
```

Comment:
I think I fixed this in the data https://www.wikidata.org/wiki/Q131917174

Reply:
it's probably going to take a few days to disappear from cache - well spotted